src/components/application_manager/src/commands/mobile/perform_audio_pass_thru_request.cc

```diff
@@ -62,7 +62,8 @@ void PerformAudioPassThruRequest::onTimeOut() {
   LOG4CXX_AUTO_TRACE(logger_);
 
   FinishTTSSpeak();
-  CommandRequestImpl::onTimeOut();
+  const std::string info = awaiting_ui_response_ ? "UI component does not respond":"";
```

Comment:
@pold500 @dcherniev 
This function is called in case of UI timed out or TTS timed out. It is common for both interfaces and at this point of view it is possible awaiting_ui_response to be false inside. This may be verified by some test script of this CRQ - for missing UI part and missing TTS part.